data/json/items/comestibles/sandwich.json

```
@@ -335,6 +335,25 @@
     "fun": 14,
     "vitamins": [ [ "vitA", 4 ], [ "vitC", 5 ], [ "calcium", 12 ], [ "iron", 24 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "lobster_roll",
+    "name": { "str": "lobster roll", "str_pl": "lobster rolls" },
+    "weight": "201 g",
+    "color": "pink",
+    "spoils_in": "1 day",
+    "container": "wrapper",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "calories": 500,
```

This might, hopefully, fix the jankiness with the comestible test.
```suggestion
    "calories": 850,
```